Entities/Vehicles/Common/VehicleAttachment.as

```diff
@@ -25,10 +25,12 @@ void GetButtonsFor(CBlob@ this, CBlob@ caller)
 				CBlob@ occBlob = ap.getOccupied();
 				if (occBlob !is null) //detach button
 				{
-					if (this.isOnGround() && occBlob.getName() != "mounted_bow")	  // HACK:
+					if (this.isOnGround() || occBlob.getName() == "mounted_bow")	  // HACK:
```

Comment:
Pr works fine, but IMO just remove the getName check, does not make sense to have the bow as the only thing you can drop while flying/moving.